### PR TITLE
Update keyword used for the frontpage

### PR DESCRIPTION
**Pull request by dwick**
:eyeglasses: @zeantsoi 

looks like i missed a spot :(. i'll make sure existing stuff is updated.

### CODE REVIEW

File: reddit_adzerk/adzerkpromote.py [hunk 1]
```diff
@@ -284,11 +284,15 @@ def update_flight(link, campaign, az_campaign):
...
         delayed_start = campaign.start_date
 
-    keywords = campaign.target.subreddit_names
+    keywords = set(campaign.target.subreddit_names)
 
-    # Don't allow nsfw links to show up on the frontpage
-    if link.over_18:
-        keywords.append('!' + Frontpage.name)
+    if Frontpage.name in keywords:
+        keywords.remove(Frontpage.name)
```
Comment (zeantsoi) [inline reddit_adzerk/adzerkpromote.py]: If the link is NSFW, we probably wouldn't want to add the `s.frontpage` keyword, right?

    if Frontpage.name in keywords:
    	# Don't allow nsfw links to show up on the frontpage
    	if link.over_18:
        	keywords.add("!s.frontpage")
    	else:
	    	keywords.remove(Frontpage.name)
	    	keywords.add("s.frontpage")

**Comment by zeantsoi** [comment]
:nail_care:

**Comment by dwick** [comment]
:haircut: good call.

**Comment by zeantsoi** [comment]
:fish:

**Pull request merged**